pbft/pbft.hpp

```diff
@@ -87,13 +87,13 @@ namespace bzn
         void do_prepared(const std::shared_ptr<pbft_operation>& op);
         void do_committed(const std::shared_ptr<pbft_operation>& op);
 
-        void handle_pbft_message(const bzn::message& json, std::shared_ptr<bzn::session_base> session);
-        bzn::message wrap_message(const pbft_msg& message, const std::string& debug_info = "");
-        bzn::message wrap_message(const audit_message& message, const std::string& debug_info = "");
+        void handle_bzn_message(const wrapped_bzn_msg& msg, std::shared_ptr<bzn::session_base> session);
+        std::string wrap_message(const pbft_msg& message, const std::string& debug_info = "");
```

Comment:
can you define a type for std::string?
perhaps reuse bzn::message and rename the json version to something else?